helm/postgres/templates/_helpers.tpl

```diff
@@ -0,0 +1,82 @@
+{{- define "mychart.getSecretValue" -}}
```

Comment:
nit: maybe we could rename this partials file to something else, so it does not conflict with the upstream? (just in case owners create it in the future)